include/triton/core/tritonserver.h

```diff
@@ -259,13 +259,31 @@ typedef enum TRITONSERVER_loglevel_enum {
   TRITONSERVER_LOG_VERBOSE
 } TRITONSERVER_LogLevel;
 
+/// Logging Formats
 ///
-/// Format of logging.
+/// The TRITONSERVER API offers two logging formats. The formats have
+/// a common set of fields but differ in how the timestamp for a log
+/// entry is represented. Messages are serialized according to JSON
+/// encoding rules by default. This behavior can be disabled by
+/// setting the environment variable TRITON_SERVER_ESCAPE_LOG_MESSAGES
+/// to "0".
 ///
-/// TRITONSERVER_LOG_DEFAULT: the log severity (L) and timestamp will be
-/// logged as "LMMDD hh:mm:ss.ssssss".
 ///
-/// TRITONSERVER_LOG_ISO8601: the log format will be "YYYY-MM-DDThh:mm:ssZ L".
+/// 1. TRITONSERVER_LOG_DEFAULT
+///
+/// <level><month><day><hour>:<min>:<sec>.<usec> <pid> <file>:<line>] <msg>
+///
+/// Example:
+///
+/// I0520 20:03:25.829575 3355 model_lifecycle.cc:441] "AsyncLoad() 'simple'"
+///
+/// 2. TRITONSERVER_LOG_ISO8601
+///
+/// <year>-<month>-<day>T<hour>:<min>:<sec>Z <level> <pid> <file>:<line>] <msg>
```

Comment:
does the format include the ISO standard `.<msec>`?

Reply:
it doesn't - the standard allows for both (IIRC) - but we do plan to consider it after these changes.